### PR TITLE
Flesh out "Our Team" section

### PR DESCRIPTION
**Pull request by andycochran**
* add Andy Cochran to Our Team
* style media object component

### CODE REVIEW

**Comment by pichot** [comment]
🎉 Welcome Andy!

**Comment by rememberlenny** [comment]
👍

File: _about/index.md [hunk 1]
```diff
@@ -66,8 +66,18 @@ Read the full text of [our charter](https://github.com/NYCPlanning/labs-charter)
...
 ## Our Team
 
-Coming Soon
+### Andy Cochran: Designer
+<div class="media-object">
+  <figure class="thumbnail">
+    <img src="https://pbs.twimg.com/profile_images/486195376077029377/U1cXHuNZ_400x400.jpeg">
+  </figure>
+  <div class="description">
+    <p>Andy joined Planning Labs in July, 2017 after implementing New York City Council's plan for digital inclusion and open government. As Design &amp; Creative Director at <a href="http://openplans.org/">OpenPlans</a>, he built collaborative mapping services and other civic engagement tools.</p>
```
Comment (IanLee1521) [inline _about/index.md]: Joining from the future?! Impressive!

Comment (konklone) [inline _about/index.md]: The sign of a true disruptor

**Comment by chriswhong** [comment]
This is the best, nerdiest, open sourciest way to announce your arrival... I love it.  Welcome aboard Andy!

**Comment by noneck** [comment]
NURDS!!!!

Congrats

**Comment by gbinal** [comment]
_Nice._   :)

**Comment by eddietejeda** [comment]
👍 👍 👍

**Pull request merged**